_about/notes/0.6.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Istio 0.6
+
+order: 95
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Custom Envoy Config**. Pilot now supports ferrying custom Envoy config to the
+proxy. TBD: doc link?
+
+## Mixer adapters
+
+- **SolarWinds**. Mixer can now interface to AppOptics and Papertrail.
+[Learn more]({{home}}/docs/reference/config/adapters/solarwinds.html)
+
+- **Redisquota**. Mixer now supports a Redis-based adapter for rate limit tracking.
+[Learn more]({{home}}/docs/reference/config/adapters/redisquota.html)
+
+- **Datadog**. Mixer now provides an adapter to deliver metric data to a Datadog agent.
+[Learn more]({{home}}/docs/reference/config/adapters/datadog.html)
+
+## Other
+
+- **Separate Check & Report Clusters**. When configuring Envoy, it's now possible to use different clusters
+for Mixer instances that are used for Mixer's Check functionality from those used for Mixer's Report
+functionality. This may be useful in large deployments for better scaling of Mixer instances.
+TBD: doc link?
+
+- **Helm Charts**. TBD: what can we say? TBD: doc link?
+
```

Comment:
I think we should encourage really early adopter to try master instead of 0.6, see https://github.com/istio/istio/issues/3895 (fixed in master), with known issue on auto side car injection: https://github.com/istio/istio/issues/3864

Reply:
The helm documentation is here: https://istio.io/docs/setup/kubernetes/helm.html

Reply:
I'll remove Helm charts from the release notes for now.